default/stringtables/en.txt

```diff
@@ -6238,6 +6241,40 @@ Opinions of Empires:
 OPINIONS_OF_OTHER_SPECIES
 Opinions of Other Species:
 
+# Species Pedia Categories
+
+LITHIC_SPECIES_CLASS
+Lithic Species
+
+LITHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia LITHIC_SPECIES_TITLE]] species:
+
+ORGANIC_SPECIES_CLASS
+Organic Species
+
+ORGANIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ORGANIC_SPECIES_TITLE]] species:
+
+PHOTOTROPHIC_SPECIES_CLASS
+Phototrophic Species
+
+PHOTOTROPHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia PHOTOTROPHIC_SPECIES_TITLE]] species:
+
+ROBOTIC_SPECIES_CLASS
+Robotic Species
+
+ROBOTIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ROBOTIC_SPECIES_TITLE]] species:
+
+SELF_SUSTAINING_SPECIES_CLASS
```

Comment:
This shouldn't have "Species" in it as it looks oddly doubled as the article type in the pedia
![cat](https://cloud.githubusercontent.com/assets/11386908/24052288/4272996a-0b35-11e7-8ae0-09f9bc94713b.png)